src/elm/interval.coffee

```diff
@@ -137,7 +137,12 @@ module.exports.Start = class Start extends Expression
   exec: (ctx) ->
     interval = @arg.execute(ctx)
     return null if !interval?
-    return interval.start()
+    start = interval.start()
+    # fix the timezoneOffset of minimum Datetime to match context offset
+    if start?.isDateTime && start.equals(MIN_DATETIME_VALUE)
+      start.timezoneOffset = ctx.getTimezoneOffset()
```

Comment:
So... this wouldn't survive a round-trip, would it?  In other words, now `start` does *not* equal `MIN_DATETIME_VALUE` anymore -- and if we were to reconstruct an interval with it, it wouldn't be identified as the min/max anymore.  I guess that's OK?

Reply:
After discussing over Slack I get it now.  No problem.